Tools/Scripts/webkitpy/style/checkers/cpp.py

```diff
@@ -1371,7 +1371,7 @@ def process_clean_line(self, line):
             self.in_enum_decl = True
             self.enum_decl_name = matched.group('identifier')
         else:
-            matched = match(expr_enum_start + r'(?P<members>.*)' + expr_enum_end + r'$', line)
+            matched = match(expr_enum_start + r'(?P<members>[^{]*)' + expr_enum_end + r'$', line)
```

Comment:
why is this needed?

Reply:
Otherwise it will greedily match the type as part of the first element: ` : uint8_t { Uno`

Reply:
I considered making `{`  required in `expr_enum_start` but there are enum declarations in the code that have `{` on the next line.